sdk/cosmos/azure-cosmos/azure/cosmos/aio/_retry_utility_async.py

```diff
@@ -35,6 +35,7 @@
 from .. import _resource_throttle_retry_policy
 from .. import _default_retry_policy
 from .. import _session_retry_policy
+from .. import _gone_retry_policy
```

Comment:
Will these changes be included in sync too?
If so - would it be breaking?

Reply:
they are also included in the sync and it shouldn't be breaking - we are fixing a gap that was present by adding this functionality. moreover, the user doesn't get to interact with the retry utility or the retry policies.
the sync changes are here if you're curious, I'll be making changes to this async fixes to have them be closer in pairity, the big exception being the async heap: https://github.com/Azure/azure-sdk-for-python/pull/22237/files